.eslintrc

```diff
@@ -27,6 +45,12 @@
       }
     ],
     "react/prop-types": "off",
-    "react/display-name": "warn"
+    "react/display-name": "warn",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ]
```

Comment:
Google-fu is failing me, where are the options for this documented? And specifically, what does this option do?

Reply:
Looks like when I cargo-culted this section, I managed to bring in a CR/LF fix: https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-delete-cr-prettier-prettier